tests/hackathon/README.md

```diff
@@ -0,0 +1,50 @@
+## What is this app/demo doing?
+
+There are banks and regulators. Banks issue transactions that can not be leaked outside of CCF. Regulators register algorithms that
```

Comment:
I think we should give a sample file for the transactions data (e.g. with 10 lines of data) and explain here that the test assigns a random country among `["US", "GB", "FR", "GR", "AU", "BR", "ZA", "JP", "IN"]` to each ingested transaction .